masterlist.yaml

```diff
@@ -1004,6 +1004,22 @@ plugins:
     url: [ 'https://www.nexusmods.com/newvegas/mods/62041/' ]
     group: *fixesGroup
 
+  - name: 'NVSE/Plugins/sr_New_Vegas_Stutter_Remover.dll'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/34832'
+        name: 'New Vegas Stutter Remover'
+    msg:
+      - <<: *obsolete
+      - <<: *useInstead
+        type: error
```

Comment:
I don't think it's necessary to raise `useInstead`'s `type` from `warn` to `error`. As such please remove the three different `type: error` lines.